resources/assets/js/components/privacy-info/PrivacyInfoLogic.vue

```diff
@@ -0,0 +1,32 @@
+<script>
```

Comment:
I'm not sure about having this small amount of code in a separate file, it looks like starting a bad practice to me when we end up with a bunch of files like in the admin package.

I think that we should stick to one file and if we end up with very large logic then we should consider a new file or maybe reconsider the whole component and why we end up with so much code in it.